backend/libs/artemislib/setup.py

```diff
@@ -27,7 +27,7 @@
         # separately because it contains platform-dependent pre-compiled code
         "pyjwt[crypto]",
         "requests",
-        "urllib3<2",  # https://github.com/boto/botocore/issues/2926
+        "urllib3",
```

Comment:
It looks like this change was made in `backend/libs/artemislib/setup.py` rather than `backend/lambdas/events/metadata_handler/setup.py` - the `setup.py` for `metadata_handler` still seems to have `urllib3<2`.

We might not _want_ to mess with `artemislib` in an otherwise-unrelated PR. I was just suggesting testing this change in the new component.